### PR TITLE
Add a few color sets (conventions) found online

### PR DESCRIPTION
**Pull request by jimallman**
I've adapted a few of the wire-color conventions listed in [this discussion](https://community.vcvrack.com/t/choosing-cable-colors-in-vcv-1-0/4155).

These will appear as options in WM-101 *if* there isn't already a `WM-101.json` file for this user. The existing default colors will then be active, just as before.

Ideally we'd have a feature to add/merge these shared color sets with the user's saved settings.

### CODE REVIEW

**Comment by david-c14** [comment]
I thought that collections could already be named.

**Comment by jimallman** [comment]
> Is the intention here to gather a name when the collection is created, rather than having to rename it afterwards?

Yes, that was my thought. Personally, I didn't realize they could be named until I stumbled on the UI later. I'm happy to revert those changes if you don't like the jump to the collections list.

**Comment by jimallman** [comment]
> I'm not clear why you've increased the item height.

I'm hoping to show the single-color descriptions (which don't exist yet) in the collections list. So you could move the mouse over the color blocks and see at a glance how the collection describes each color (e.g. "Modulation", "V/Octave"). The current color blocks seem too small to comfortably do this, so I tried stretching them a bit.

You can see the tooltips now when hovering over the color-strip for each collection -- I just have placeholder text ("FOO-#") there now while I work out how to store and retrieve the color descriptions.

File: src/WM1.cpp [hunk 1]
```diff
@@ -319,6 +322,50 @@ struct ColorCollectionButton : EventWidgetButtonBase {
...
 	}
+
+	void onEnter(const event::Enter &e) override {
+		OpaqueWidget::onEnter(e);
+		//e.consume(this);
+	}
+	void onHover(const event::Hover &e) override {
+		if (!tooltip) {
+			counter++;
+			SubTooltip *stt = new SubTooltip();
```
Comment (david-c14) [inline src/WM1.cpp]: I like the re-use of the SubTooltip :thumbsup:

**Comment by jimallman** [comment]
> I'm hoping to show the single-color descriptions (which don't exist yet) in the collections list. So you could move the mouse over the color blocks and see at a glance how the collection describes each color (e.g. "Modulation", "V/Octave").

Now I'm having second thoughts about this idea. After making [this mockup](https://github.com/david-c14/SubmarineFree/issues/80#issuecomment-602056923), I'm realizing it would be very quick for a user to try out an unfamiliar collection and see the color descriptions in WM-101's default view. So maybe this is just extra complexity...

**Comment by david-c14** [comment]
No problem.  Have a think about it.  In the meantime, I'm going to release what I've got.  You can checkout the develop branch to see how it works for you.

**Comment by jimallman** [comment]
OK, I figured out how to keep pace with you in my forked repo, and the `develop` branch works for me. (I do like the loading/saving of single collections!)
I've reverted the most recent commit, so this PR is now just the added collections that are installed by default, plus the prompt-for-name when saving the current colors as a collection.

**Pull request merged**